src/Lykke.AlgoStore.Service.Logging/Startup.cs

```diff
@@ -2,31 +2,52 @@
 using System.ComponentModel.DataAnnotations;
 using AutoMapper;
 using Common.Log;
+using Lykke.AlgoStore.Security.InstanceAuth;
+using Lykke.AlgoStore.Service.Logging.Filters;
 using Lykke.AlgoStore.Service.Logging.Settings;
 using Lykke.Common.Api.Contract.Responses;
 using Lykke.Sdk;
+using Lykke.SettingsReader;
 using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace Lykke.AlgoStore.Service.Logging
 {
     public class Startup
     {
+        public IConfigurationRoot Configuration { get; }
         private ILog _log;
 
+        public Startup(IHostingEnvironment env)
+        {
+            var builder = new ConfigurationBuilder()
```

Comment:
I needed to set up mapper as well 
Have you tested authentication process ...
 Mapper.Initialize(cfg =>
            {
                cfg.AddProfiles(typeof(AutoMapperModelProfile));
            });